bindings/nodejs/lib/bindings.ts

```diff
@@ -99,6 +99,11 @@ const callWalletMethodAsync = (
         });
     });
 
+// Allow bigint to be serialized as hex string.
+(BigInt.prototype as any).toJSON = function () {
+    return bigIntToHex(this);
+};
```

Comment:
This works because the amount that should be string encoded will always be string encoded before, right?
If so, maybe extend the comment with that

Reply:
I'm not sure what do you mean exactly, but the bindings expect a hex encoded string for the native tokens, if you keep it as of type `bigint`, you get an error that it doesn't know what to do with a `bigint` since it calles `JSON.stringify` on the object that contains the amount.
for example 
```javascript
let a = {a: BigInt(10)}
JSON.stringify(a)
```
will output 
```
Uncaught TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at <anonymous>:1:6
```
but if the `.toJSON` is added to `BigInt` then it will serialize it to hex string as defined in `bigIntToHex`

Reply:
Yeah this part I understand, but for the amount we also have bigint and there it should be serialized as string, so this distinction is made earlier?

Reply:
Done, I added extra comment.